docs/source/getting-started/examples/polars-quantiles.rst

```diff
@@ -0,0 +1,127 @@
+Polars API
+==========
+
+The Polars API for OpenDP is under development and subject to change.
+We are currently focused on supporting a particular use case,
+but we plan to expand the API until it is the preferred interface
+for both Python and R.
+
+The OpenDP Polars API leverages `Polars <https://docs.pola.rs/>`_:
+You'll use Polars methods like ``group_by``, ``agg``, ``alias``, and ``col`` to construct
+most of your query, and only use ``dp`` extensions when needed.
+
+This example demonstrates how to construct DP aggregate statistics, including quantiles.
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. literalinclude:: code/polars-quantiles.rst
+            :language: python
+            :start-after: init
+            :end-before: /init
+
+We need to understand the structure of our private data before we can apply differential privacy,
+but we usually shouldn't or can't look at the private data as we prepare our analysis.
+For this example, we'll keep the data simple:
+
+* ``grouping-key``: the grouping key, an integer between 1 and 5
+* ``noisy-key``: most of the time, equal to ``grouping-key``, but some rows are set to 1, and some are set to 5.  
+* ``ones``: a constant value, the float 1.0
+
+Our first step is to represent this as a domain:
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. literalinclude:: code/polars-quantiles.rst
+            :language: python
+            :start-after: init-domain
+            :end-before: /init-domain
+
```

Comment:
Separate docs page would be useful for explaining the descriptors on frame domains.